tsconfig.json

```diff
@@ -76,7 +76,7 @@
     "forceConsistentCasingInFileNames": true,            /* Ensure that casing is correct in imports. */
 
     /* Type Checking */
-    "strict": true,                                      /* Enable all strict type-checking options. */
+    "strict": false,                                     /* Enable all strict type-checking options. */

```

Comment:
qual motivo de ter mudado o `strict`

Reply:
Porque estava me forçando a deixar as propriedades da entidades como opcionais ou permitir que o valor fosse undefined

Reply:
No onboard eu trabalharia com strict false tbm. A maioria dos projetos e o template está assim.